misk-core/src/main/kotlin/misk/compress/DeflateCompressionCodec.kt

```diff
@@ -0,0 +1,52 @@
+package com.squareup.misk.compress
+
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.util.zip.Deflater
+import java.util.zip.Deflater.DEFAULT_COMPRESSION
+import java.util.zip.DeflaterOutputStream
+import java.util.zip.InflaterOutputStream
+
+/**
+ * A {@link CompressionCodec} implementation for the deflate compression format. It uses the
+ * implementation provided in the standard {@code java.util.zip} package.
+ *
+ * Cribbed from SQ/repos/java/browse/compression.
```

Comment:
Not particularly useful for OSS

Reply:
Whoopsies, caught in the act. This was a skim PR first :)